src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -713,22 +713,57 @@ public static partial class Buffer
         public unsafe static void MemoryCopy(void* source, void* destination, ulong destinationSizeInBytes, ulong sourceBytesToCopy) { }
         public static void SetByte(System.Array array, int index, byte value) { }
     }
-    public readonly partial struct Byte : System.IComparable, System.IComparable<byte>, System.IConvertible, System.IEquatable<byte>, System.ISpanFormattable, System.IBinaryInteger<byte>, System.IMinMaxValue<byte>, System.IUnsignedNumber<byte>
+    public readonly partial struct Byte : System.IAdditionOperators<byte, byte, byte>, System.IAdditiveIdentity<byte, byte>, System.IBinaryInteger<byte>, System.IBinaryNumber<byte>, System.IBitwiseOperators<byte, byte, byte>, System.IComparable, System.IComparable<byte>, System.IComparisonOperators<byte, byte>, System.IConvertible, System.IDecrementOperators<byte>, System.IDivisionOperators<byte, byte, byte>, System.IEqualityOperators<byte, byte>, System.IEquatable<byte>, System.IFormattable, System.IIncrementOperators<byte>, System.IMinMaxValue<byte>, System.IModulusOperators<byte, byte, byte>, System.IMultiplicativeIdentity<byte, byte>, System.IMultiplyOperators<byte, byte, byte>, System.INumber<byte>, System.IParseable<byte>, System.IShiftOperators<byte, byte>, System.ISpanFormattable, System.ISpanParseable<byte>, System.ISubtractionOperators<byte, byte, byte>, System.IUnaryNegationOperators<byte, byte>, System.IUnaryPlusOperators<byte, byte>, System.IUnsignedNumber<byte>
     {
         private readonly byte _dummyPrimitive;
+        public const byte AdditiveIdentity = (byte)0;
         public const byte MaxValue = (byte)255;
         public const byte MinValue = (byte)0;
+        public const byte MultiplicativeIdentity = (byte)1;
+        public const byte One = (byte)1;
+        public const byte Zero = (byte)0;
+        static byte System.IAdditiveIdentity<System.Byte,System.Byte>.AdditiveIdentity { get { throw null; } }
+        static byte System.IMinMaxValue<System.Byte>.MaxValue { get { throw null; } }
+        static byte System.IMinMaxValue<System.Byte>.MinValue { get { throw null; } }
+        static byte System.IMultiplicativeIdentity<System.Byte,System.Byte>.MultiplicativeIdentity { get { throw null; } }
+        static byte System.INumber<System.Byte>.One { get { throw null; } }
+        static byte System.INumber<System.Byte>.Zero { get { throw null; } }
+        public static System.Byte Abs(System.Byte value) { throw null; }
```

Comment:
Just noting that `GenerateReferenceSource` isn't consistent in when it picks `System.Byte` vs `byte`. It should likely prefer the latter since that's the default we use in source and is unambiguous.

Reply:
Likewise noting that `GenerateReferenceSource` doesn't like `static abstracts in interfaces` today. It doesn't include the relevant `accessibility` modifier (such as `public`) and it treats operators as their IL name, rather than their C# syntax (e.g. `op_Addition` rather than `operator +`).

Reply:
Thanks for commenting on these. Please accumulate all of the `GenerateReferenceSource` issues you're encountering and log an issue in dotnet/arcade.
([some existing issues for reference](https://github.com/dotnet/arcade/issues?q=is%3Aissue+Microsoft.DotNet.GenAPI+label%3Aarea-Infrastructure-libraries))